applications/tari_validator_node/src/dry_run_transaction_processor.rs

```diff
@@ -56,10 +56,11 @@ use tari_validator_node_rpc::{
     proto::rpc::VnStateSyncResponse,
 };
 use thiserror::Error;
+use tokio::task;
 
 use crate::{p2p::services::template_manager::TemplateManager, payload_processor::TariDanPayloadProcessor};
 
-const LOG_TARGET: &str = "tari::validator_node::dry_run_transaction_processor";
+const LOG_TARGET: &str = "tari::dan::validator_node::dry_run_transaction_processor";
```

Comment:
Should we make it the default across codebase ?

Reply:
This is how it is currently set up in the `log4rs.yaml`s so `tari::xxxx` will not emit logs (other than specific configs for the L1 tari repo, tari::comms, etc). The current standard for this repo is `tari::dan::xxxx`.